layout.md

```diff
@@ -0,0 +1,78 @@
+```sh
+/sphere_root/
+ ├── .sphere/
+ │   ├── identity
+ │   ├── version  # Last rendered version of the root sphere
+ │   ├── slugs    # Backward mapping of slugs; flat directory where slugs have '∕' in place of '/'
+ │   │   ├── foo -> ../../foo.subtext
+ │   │   └── bar∕baz -> ../../bar/baz.subtext
+ │   ├── storage/ # Storage folder distinguishes the root sphere
```

Comment:
So this also stores the content for the peers?

Reply:
Yup

Reply:
I should mention the contents of `layout.md` are my personal notes, not documentation, and the structure depicted is not representative of the actual implementation.

Reply:
Here's the note to please add it to some form of real documentation b/c it seems really really helpful. :D